### ISSUE TITLE
Plotting occurrence distributions on map

### ISSUE THREAD

**Issue by chfleming** [ctmm-initiative/ctmmweb#95]
It appears that you can plot the home-range distribution atop the map layer, but not the occurrence distributions.

**Comment by xhdong-umd** [comment]
Yes, I didn't code it because I thought the occurrence plot have much more details which will interfere with map details. I am not sure if that will be useful. Also the homerange are just polygons and easy to overlay on map, I am not sure about occurrence plot which looks much more complicated.

**Comment by chfleming** [comment]
We have students requesting this feature. The home-range objects and occurrence objects should export the same way.

**Comment by chfleming** [comment]
We also have some managers using this functionality on the command line too.

**Comment by markyxe** [comment]
This feature would be very useful, so thanks for considering it. It would also be useful to be able to save the occurrence distributions as a shapefile. Thank you.

**Comment by rmccabe2** [comment]
I would also be interested in having the occurrence objects export as a shapefile. Is this a feature that could be available in the ctmmweb app within the next few months?

**Comment by xhdong-umd** [comment]
@chfleming , So the exported occurrence only have some polygons, 

```r
spdf <- SpatialPolygonsDataFrame.UD(occur_list_sub2[[2]])
plot(spdf)
```

<img width="191" alt="Screen Shot 2019-09-10 at 7 22 56 PM" src="https://user-images.githubusercontent.com/25039897/64657217-6b7db600-d400-11e9-9db2-1d942e38833c.png">

and the details in plot are not available in export, right?

```r
plot(occur_list_sub2[[2]])
```

<img width="285" alt="Screen Shot 2019-09-10 at 7 23 13 PM" src="https://user-images.githubusercontent.com/25039897/64657231-79cbd200-d400-11e9-9d96-7a584d34497b.png">

Should I use some more confidence intervals, or just default option is enough?

**Comment by chfleming** [comment]
AKDEs have 3 polygons for the CIs. Occurrence distributions are themselves CIs and by default there is only one polygon exported.

People are probably more interested in the raster export.

**Comment by rmccabe2** [comment]
If both options (raster and shapefile) could be available that would be great! I would like to identify the land cover types within the polygons.

**Comment by xhdong-umd** [comment]
Export occurrence to shapefile, raster, tiff (same with home range) is added.

To add occurrence to map need more changes. I probably need to change how each layers are named as there will be points/home range/occurrence layers in the map. Previously single model was renamed by individual names without model postfix, but that might need to change as this will cause too much problems in naming.
